templates/terraform/schema_property.erb

```diff
@@ -86,6 +90,15 @@
         <% end -%>
       },
   <% end -%>
+  <% if property.is_set -%>
+    <% if !property.set_hash_func.nil? -%>
```

Comment:
should this be `unless property.set_hash_func.nil?`

Reply:
you can't have else/elsif statement with an unless.